src/web/templates/admin/pairings/safety_mode_modal.html

```diff
@@ -12,60 +12,101 @@ <h5 class="text-danger">
                         {{ _('Warning: you are trying to execute an operation forbidden by the FIDE rules.') }}
                     </h5>
                 {% endif %}
-                <h6>
-                    {{
-                        _('Displayed round: %(round)d', round=admin_round) ~ ' - ' ~
-                        _('Current round: %(round)d', round=admin_tournament.current_round)
-                    }}
-                </h6>
-
-                <p>
-                    {% if required_mode == 'FIDE_INCOMPATIBLE' %}
+                {% if action == 'FULL_UNPAIRING' %}
+                    <p>
                         {{ _(
-                            'The FIDE rules stipulate that only the pairings of the round '
-                            '<b>prior</b> to the current one may be modified. Changes to '
-                            'pairings previous to this should be made at the <b>end</b> of '
-                            'the tournament, <b>after</b> the rankings have been published, '
-                            'and solely for the computation of the FIDE ratings.'
+                            'Unpairing a published round can be very inconvenient for players. '
+                            'In addition, it causes a probable loss of data, as all the '
```

Comment:
Additionally, you will lose a data associated with this round, including results and manual adjustments.